examples/cloudflare-pages/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "remix-cf-pages-template-hmr",
+  "type": "module",
+  "scripts": {
+    "build": "remix build",
+    "predev": "remix build",
+    "dev": "concurrently \"npm:dev:*\"",
+    "dev:remix": "remix dev",
+    "dev:wrangler": "wrangler pages dev public --port 3000 -b ENVIRONMENT=development SESSION_SECRET=foobar PUBLIC_STOREFRONT_API_TOKEN=3b580e70970c4528da70c98e097c2fa0 PUBLIC_STOREFRONT_API_VERSION=2023-01 PUBLIC_STORE_DOMAIN=hydrogen-preview.myshopify.com"
```

Comment:
I was trying `--env` and it was not binding the variables. Anybody knows is if there is an easier way?